### PR TITLE
CLI fixes regarding parameters

### PR DESCRIPTION
**Pull request by axic**
Fixes typos/bugs introduced by #189 and #205.

### CODE REVIEW

**Comment by jwasinger** [comment]
```
bin/evm2wasm.js 0x600160020200 --trace
```

Is this ever being ran? Gives me
```
Error: TypeError: "value" argument must not be a number
```

**Comment by axic** [comment]
It was run unless someone removed the travis task.

**Comment by jwasinger** [comment]
```

/home/travis/.travis/job_stages: line 58: bin/evm2wasm.js: No such file or directory
```

**Comment by axic** [comment]
The disturbing fact is that we assumed those subshells have `set -e` on.

**Comment by axic** [comment]
Circleci claims it is on by default: https://circleci.com/docs/2.0/configuration-reference/#default-shell-options

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ before_install:
     git submodule update --init
     if [ $LANG = js ]; then
       ./tools/wabt/scripts/travis-before-install.sh
-      cd ./tools/wabt/ && make
+      (cd ./tools/wabt/ && make)
```
Comment (jwasinger) [inline .travis.yml]: What does this do?

**Comment by axic** [comment]
```
/home/travis/build/ewasm/evm2wasm
Error: TypeError: "value" argument must not be a number
```

At least now executes, but this is not considered an error.

**Comment by jwasinger** [comment]
https://github.com/ewasm/evm2wasm/pull/214

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     "transcompiler"
   ],
   "dependencies": {
+    "eslint": "^4.19.1",
```
Comment (axic) [inline package.json]: Which part needed eslint?

**Comment by axic** [comment]
@jwasinger squashed some and improved the commit log. They are distinct bugfixes though.

**Pull request merged**